common/src/main/java/com/microsoft/identity/common/internal/fido/AuthFidoChallengeHandler.kt

```diff
@@ -190,9 +190,10 @@ class AuthFidoChallengeHandler (
         if (exception != null) {
             span.recordException(exception)
             span.setStatus(StatusCode.ERROR)
+            respondToChallenge(submitUrl, exception.toString(), context, span)
```

Comment:
nit: exception.toString() -> e.getMessage()?

Not sure if the first one prints out a hash code. might want to double check.

Reply:
e.getMessage() doesn't include the actual name of the exception... so I changed this to exception.javaclass.name() + ": " + exception.getMessage(): 6a7adc04646c80b529cbbcf03721528ce47aef93